x-pack/plugins/rollup/public/crud_app/index.js

```diff
@@ -61,15 +45,27 @@ routes.when(`${CRUD_APP_BASE_PATH}/:view?`, {
   template: template,
   controllerAs: 'rollupJobs',
   controller: class IndexRollupJobsController {
-    constructor($scope, $route, $http) {
+    constructor($scope, $route, $injector) {
       // NOTE: We depend upon Angular's $http service because it's decorated with interceptors,
       // e.g. to check license status per request.
-      setHttpClient($http);
+      setHttp($injector.get('$http'));
```

Comment:
I prefer the HttpClient here as it does not indicate that the client is $http.

Reply:
I was thinking that because this is a temporary hack, there's value in communicating that this code is specifically intended to support the use of $http, and not something meant to be used for any XHR lib. For example once we migrate to kfetch all of this code will be deleted. This pattern of creating `set${AngularModule}` modules can also scale to other Angular services which we want to expose to React code, which we intend to eventually remove. How do you feel about this?